packages/lit-dev-content/samples/tutorials/working-with-lists/04/before/my-element.ts

```diff
@@ -0,0 +1,50 @@
+import {html, css, LitElement} from 'lit';
+import {customElement} from 'lit/decorators.js';
+// TODO: Import directives
+
+@customElement('my-element')
+class MyElement extends LitElement {
+  static styles = css`
+    /* playground-fold */
+    :host {
+      display: block;
+      width: 400px;
+      height: 400px;
+    }
+    #board {
+      display: grid;
+      grid-template-columns: repeat(8, 12.5%);
+      grid-template-rows: repeat(8, 12.5%);
+      border: 2px solid #404040;
+      box-sizing: border-box;
+      height: 100%;
+    }
+    .square {
```

Comment:
Remove `.square` completely here as this isn't included in the instructions and breaks the step.